### PR TITLE
Version 3 variant

### PR DESCRIPTION
**Pull request by cyx**

### CODE REVIEW

File: lib/commands/create.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ function create(context) {
         }
       }
     }).then(function(build) {
-      request.get(build.stream_url).pipe(process.stderr);
```
Comment (tt) [inline lib/commands/create.js]: So @friism used the wrong attribute?

Comment (cyx) [inline lib/commands/create.js]: The shorter one, I guess it's just now that the decision was made to choose `output_stream_url` instead.

**Comment by cyx** [comment]
Just tested with the other PR and this definitely works. :)

File: lib/commands/create.js (hunk 1 above)
Comment (friism) [inline lib/commands/create.js]: @tt this was written before the API re-arrangement

**Pull request merged**